src/main/java/com/dropbox/core/android/AuthActivity.java

```diff
@@ -581,4 +694,46 @@ private String createStateNonce() {
         }
         return sb.toString();
     }
+
+    private String createPKCEStateNonce() {
+        return String.format("oauth2code:%s:%s:%s",
+                             mPKCEManager.getCodeChallenge(),
+                             DbxPKCEManager.CODE_CHALLENGE_METHODS,
+                             mTokenAccessType.toString());
+    }
+
+    private enum TokenType {
+        OAUTH2("oauth2"),
+        OAUTH2CODE("oauth2code:");
+
+        private String string;
+
+        TokenType(String string) {
+            this.string = string;
+        }
+
+        @Override
+        public String toString() {
+            return string;
+        }
+    }
+
+    private class TokenRequest extends AsyncTask<Void, Void, DbxAuthFinish> {
```

Comment:
Rename this to `TokenRequestAsyncTask` so it's clear that this is an `AsyncTask`

Reply:
Done